HIPs/hip-1.md

```diff
@@ -0,0 +1,271 @@
+
+The best way to get client implementers to review your HIP is to present it on an AllCoreDevs call. You can request to do so by posting a comment linking your HIP on an [AllCoreDevs agenda GitHub Issue].  
+
+The AllCoreDevs call serve as a way for client implementers to do three things. First, to discuss the technical merits of HIPs. Second, to gauge what other clients will be implementing. Third, to coordinate HIP implementation for network upgrades.
+
+These calls generally result in a "rough consensus" around what HIPs should be implemented. This "rough consensus" rests on the assumptions that HIPs are not contentious enough to cause a network split and that they are technically sound.
+
+:warning: The HIPs process and AllCoreDevs call were not designed to address contentious non-technical issues, but, due to the lack of other ways to address these, often end up entangled in them. This puts the burden on client implementers to try and gauge community sentiment, which hinders the technical coordination function of HIPs and AllCoreDevs calls. If you are shepherding an HIP, you can make the process of building community consensus easier by making sure that [the Handshake Magicians forum] thread for your HIP includes or links to as much of the community discussion as possible and that various stakeholders are well-represented.
+
+*In short, your role as the champion is to write the HIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea.* 
+
+### HIP Process 
+
+Following is the process that a successful non-Core HIP will move along:
+
+```
+[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ FINAL ]
+```
+
+Following is the process that a successful Core HIP will move along:
+
+```
+[ IDEA ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
+```
+
+Each status change is requested by the HIP author and reviewed by the HIP editors. Use a pull request to update the status. Please include a link to where people should continue discussing your HIP. The HIP editors will process these requests as per the conditions below.
+
+* **Idea** -- Once the champion has asked the Handshake community whether an idea has any chance of support, they will write a draft HIP as a [pull request]. Consider including an implementation if this will aid people in studying the HIP.
+  * :arrow_right: Draft -- If agreeable, HIP editor will assign the HIP a number (generally the issue or PR number related to the HIP) and merge your pull request. The HIP editor will not unreasonably deny an HIP.
+  * :x: Draft -- Reasons for denying draft status include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the [Handshake philosophy](https://github.com/Handshake/wiki/wiki/White-Paper#philosophy).
```

Comment:
[handshake philosophy] link broken -- do we have a philosophy? maybe https://hsd-dev.org/files/handshake.txt